Move donate to separate page (#584)
deltachat/deltachat-pages: en/donate.md

---
title: Donate Money
lang: en
---

> 

Reviewer: ```suggestion

# Donate Money
```

we have a headline on virtually all pages, to match the layout, we should add one here as well.

---
title: Donate Money
lang: en
---

# Donate Money

> Help DeltaChat develop and grow as an independent project serving the people who use it:

Reviewer: ```suggestion
Help Delta Chat develop and grow as an independent project serving the people who use it:
```

we are [not strict](https://github.com/deltachat/interface/issues/16) with the spelling of "Delta Chat", however, within a context it should be consistent (and on delta.chat we use "Delta Chat", modulo errors :)

## Euros

*Please always specify the account holder 'Björn Petersen'*

__IBAN__: DE86100777770428658900

> __SWIFT__: BIC NORSDE51XXX

Reviewer: ```suggestion
__SWIFT/BIC__: NORSDE51XXX
```

i think, this is more correct, "SWIFT" and "BIC" are used mostly synonym in this context; at least the "SWIFT: BIC ..." seems wrong as it implies somehow that the SWIFT-code starts with BIC.

## PayPal or credit card

[paypal.me/deltachat](https://paypal.me/deltachat/20)

## Liberapay

[liberapay.com/delta.chat](https://liberapay.com/delta.chat/)

## Open Collective

For one-time or recurring donations: [opencollective.com/delta-chat/donate](https://opencollective.com/delta-chat/donate)

## Bitcoin

Wallet [bc1qt3xhfvwmdqvxkk089tllvvtzqs8ts06u3u6qka](bitcoin:bc1qt3xhfvwmdqvxkk089tllvvtzqs8ts06u3u6qka)

> * * *

Reviewer: although `<hr>` is defined as a "thematically semantic break" in html5 - it is still a not-so-nice-horizontal line in practise. i would just remove this :)

Author: Done.
